composer/metrics/nlp.py

```diff
@@ -265,11 +265,22 @@ def replace_underscore(text: str) -> str:
 
         return white_space_fix(remove_articles(handle_punc(lower(replace_underscore(answer))))).strip()
 
-    def update(self, outputs: List[str], labels: List[List[str]]):
+    def update(self, batch, outputs: List[str], labels: List[List[str]]):
+        cot_delimiter = batch['cot_delimiter']
         for sample_output, sample_labels in zip(outputs, labels):
-            cleaned_sample_output = self.normalize_answer(sample_output)
+            if cot_delimiter is not None and len(cot_delimiter) > 0:
+                # in chain of thought, the final answer comes after the
```

Comment:
Capitalize first letter of comments.
```suggestion
                # In chain of thought, the final answer comes after the
```